### PR TITLE
[2.4] Add warning when the same admin in project.yml has different role

### PR DESCRIPTION
**Pull request by IsaacYangSLA**
### Description

By design, provisioners can add clients or admins in project.yml and do provision to get additional client/admin startup kits.  The certs and private keys of existing participants are kept the same so that there is no need to re-distribute those existing startup kits.

When the role of existing admin changes in project.yml, its cert/private key and thus the role in subject's unstructured_name, do not change.  This causes confusion as provisioners may think a new set of cert/private key is generated.

This PR adds a warning during provisioning time so that provisioners know what to do to generate the new startup kits.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [x] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/lighter/impl/cert.py [hunk 1]
```diff
@@ -87,6 +87,17 @@ def _build_write_cert_pair(self, participant, base_name, ctx):
             pri_key = serialization.load_pem_private_key(
                 self.persistent_state[subject]["pri_key"].encode("ascii"), password=None, backend=default_backend()
             )
+            if participant.type == "admin":
```
Comment (yhwen) [inline nvflare/lighter/impl/cert.py:90]: What's the reason we only do this for the "admin" type?

Comment (IsaacYangSLA) [reply nvflare/lighter/impl/cert.py:90]: Only admin certs have unstructured name, which contains the 'role' information, such as project_admin.

**Review by yhwen — APPROVED** [review]
LGTM

**Pull request merged**